includes/view/ShiftCalendarRenderer.php

```diff
@@ -222,14 +222,14 @@ private function renderTick($time, $label = false)
                 return div('tick day');
             }
             return div('tick day', [
-                date('m-d<b\r />H:i', $time)
+                date(__('m-d').'<br>'.__(' H:i'), $time)
```

Comment:
There was a good reason for using `\r` here as the `r` gets interpreted by the date formatter.
Also instead of using `' H:i'` as the translation string the space before it should be added after the break to make it better translatable (and circumvents double translations, ones with a space and once without it).

The clearer way would be to use `date(__('m-d'), $time) .'<br>'.date(__('H:i'), $time)` here (as  the whitespace can be omitted)